runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -58,7 +72,8 @@ native2InterpJavaUpcall0(J9UpcallMetaData *data, void *argsListPointer)
 I_32 JNICALL
 native2InterpJavaUpcall1(J9UpcallMetaData *data, void *argsListPointer)
 {
-	return (I_32)0;
+	UDATA returnValue = native2InterpJavaUpcallImpl(data, argsListPointer);
+	return (I_32)returnValue;
```

Comment:
Casts should change signed-ness or size but not both at once.
```
	U_64 returnValue = native2InterpJavaUpcallImpl(data, argsListPointer);
	return (I_32)(I_64)returnValue;
```

Reply:
PR is created at https://github.com/eclipse-openj9/openj9/pull/15900 for the fixes.